lib/client/api.go

```diff
@@ -187,8 +187,19 @@ func (c *Config) ProxySpecified() bool {
 type TeleportClient struct {
 	Config
 	localAgent *LocalKeyAgent
+
+	OnShellCreated ShellCreatedCallback
+
+	// ExitMsg (if set) will be printed at the end of the SSH session
+	ExitMsg string
 }
 
+// This callback can be supplied for every teleport client. It will be called
```

Comment:
Change the start of the sentence to `ShellCreatedCallback can be supplied...` to comply with code guidelines